doc/api/process.md

```diff
@@ -1775,6 +1784,381 @@ Will generate an object similar to:
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The name of the performance entry.
+
+#### performanceEntry.startTime
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp marking the starting time of the
+Performance Entry.
+
+#### performanceEntry.entryType
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The type of the performance entry. Current may be one of: `'node'`, `'mark'`,
+or `'measure'`.
+
+### Class: PerformanceFrame
+<!-- YAML
+added: REPLACEME
+-->
+
+A subclass of `PerformanceEntry` that provides timing metrics specific to the
+Node.js event loop.
+
+#### performanceFrame.duration
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The total number of milliseconds that have passed in the current loop.
+
+#### performanceFrame.prior
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The total number of milliseconds passed for the prior iteration of the loop.
+
+### Class: PerformanceNodeTiming
+<!-- YAML
+added: REPLACEME
+-->
+
+A subclass of `PerformanceEntry` that provides timing metrics specific to the
+Node.js runtime.
+
+#### performanceNodeTiming.arguments
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js command-line
+arguments processing completed.
+
+#### performanceNodeTiming.bootstrapComplete
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js process
+completed bootstrap.
+
+#### performanceNodeTiming.duration
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The total number of milliseconds that the Node.js process has been running.
+
+#### performanceNodeTiming.initialize
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js platform was
+initialized.
+
+#### performanceNodeTiming.inspectorStart
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js inspector
+agent completed startup.
+
+#### performanceNodeTiming.loopExit
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js event loop
+exited.
+
+#### performanceNodeTiming.loopStart
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp at which the Node.js event loop
+started.
+
+### Class: process.PerformanceObserver(callback)
+
+* `callback` {Function}
+
+`PerformanceObserver` objects provide notifications when new
+`PerformanceEntry` instances have been added to the Performance Timeline.
+
+```js
+const obs = new process.PerformanceObserver((list, observer) => {
+  console.log(list.getEntries());
+  observer.disconnect();
+});
+obs.observe({ entryTypes: ['mark'], buffered: true });
+
+process.performance.mark('test');
+```
+
+Because `PerformanceObserver` instances introduce their own additional
+performance overhead, instances should be left subscribed to notifications
```

Comment:
It may be my mediocre English, but "instances should be left subscribed to notifications indefinitely" and "Users should disconnect observers as soon as they are no longer needed" seem contradictory. Should it be "should **not** be left"?